license.txt

```diff
@@ -0,0 +1 @@
+Copyright (c) Electric Imp This file is licensed under the MIT License http://opensource.org/licenses/MIT
```

Comment:
What's this file for?

Anyway, it's incorrect, see https://electricimp.com/docs/libraries/submissions/